backend/src/controllers/user.js

```diff
@@ -0,0 +1,49 @@
+const userService = require('../services/user');
+
+async function buyStock(req, res) {
+  const {
+    stockSymbol, sharesBought, price,
+  } = req.query;
+
+  const { id } = req.query;
+
+  const { status, json } = await userService
+    .buyStock(stockSymbol, sharesBought, price, id);
+
+  res.status(status).json(json);
+}
+
+async function sellStock(req, res) {
+  const {
+    stockSymbol, sellingAmount, price,
```

Comment:
Should we just hit the AlphaVantage API from here to get the buy/sell price? Otherwise we need to hit the API on the frontend, then pass it to this endpoint.

Reply:
My argument for this was that the user would be buying/selling what they see.
E.g. if the price does change while the query is going through, they might buy at a different price compared to what they saw on the frontend

Reply:
Fair enough